docs/uefi-db.md

```diff
@@ -0,0 +1,203 @@
+---
+title: UEFI Secure Boot Certificates
+---
+
+## Executive Summary
+
+Very soon a certificate we use for signing boot media will expire. We recently found out that Microsoft will not sign updated boot media with the old key, and that **at least one tier-1 OEM** is not going to be shipping the expired key on new hardware. This means that existing install media may not boot on some new laptop, desktop and server devices, and that future updates to boot packages may not boot on old hardware.
```

Comment:
`Very soon` doesn't age well.  I'd put the specific dates.
Like:

> A certificate used for signing boot media expires on XX-YY-ZZZZ

Furthermore drop "recently"